exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,34 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  }
+  return false;
+}
```

Comment:
Hi Humail, I really like your use of IF statement in this whole exercise including the other functions too whereas you could have just done a 'return' with the condition but writing an IF statement makes it pretty clear for other users in terms of readability.

Reply:
nice job Humail. your code is clean and organized and I always enjoy your work

Reply:
Thank you Azan and Maziar. Really appreciate your comments. Thank you for the reviews.

> nice job Humail. your code is clean and organized and I always enjoy your work



> Hi Humail, I really like your use of IF statement in this whole exercise including the other functions too whereas you could have just done a 'return' with the condition but writing an IF statement makes it pretty clear for other users in terms of readability.

Reply:
Great work Humail as usual .